### ISSUE TITLE
Modify Concept: Make "Stars" the preferred label of 1634

### ISSUE THREAD

**Issue by msdemlei** [astrothesaurus/UAT#327]
**Name the concept in question**

http://astrothesaurus.org/uat/1634

**Describe the change**

The current preferred label of 1634 is "Stellar types".  That, to me, is not a concept many people would associate with the concept "Star"; but that is actually an alternative label.  

People, on the other hand, will look for "Star".  In the VO, that's the second most popular subject keyword, second only to "Galaxy" (3491 to 3527, so it's a close call).

Also, if you ask just about anyone what the common wider term is between early-type-stars, field-stars, late-type-stars, protostars, low-mass-stars, variable-stars, multiple-stars, stellar-spectral-types, novae, massive-stars, supernovae, intermediate-type-stars, stellar-evolutionary-types, emission-line-stars, chemically-peculiar-stars, and solar-analogs -- I'm sure >95% of those who say anything will say "Star".  And I'd be willing to bet if you went around and asked 100 astronomers, you wouldn't hear "Stellar types" a single time

As to what is to happen with "Stellar types" I'm not sure; my take would be that "Stellar Types" is semantially too far removed from "Star" to be a proper alternative label.  But then I can't say what the intention of the concept was in the first place.  I guess once one has suffienctly precise definitions of the two concepts, it will be quite obvious what to do.  

Oh, and while touching the concept I'd say we should add a definition.  What about:  "A  gravitationally bound object either maintaining fusion reactions or having once done that".  That excludes brown drawfs, but it would include at least white dwarfs and pulsars.  It excludes clouds (which is good) but excludes young stellar objects before fusion ignition (which is not so good; but then: we'll have to draw the line somewhere between cloud and star).

**Comment by BartlettAstro** [comment]
The An Etymological Dictionary of Astronomy and Astrophysics definition of a star is 
"A huge mass of hot gas whose radiation is provided by its internal thermonuclear reactions. A star represents a hydrodynamic equilibrium between two opposing forces, the inward gravitational force, which is attempting to make the mass collapse and the pressure caused by the generation of nuclear energy. Below a certain mass (0.08 solar masses), the central pressures and temperatures are insufficient to trigger the hydrogen fusion, brown dwarf. Stars have a variety of masses and sizes. Massive stars are less common than low-mass stars initial mass function. Star formation results from gravitational collapse of molecular clouds fragmentation; pre-stellar core protostar accretion. After leaving the main sequence, they pass through several evolutionary stages (e.g., red giant, supergiant, white dwarf, supernova, neutron star) depending on their initial masses. 
The term star is sometimes loosely applied to objects that do not comply with the above specifications, but are evolutionary products of stars, such as neutron stars and white dwarfs. For ancient civilizations a star was anything appearing in the night sky, apart from perhaps the Moon."

It does not define Stellar types.

@katieefrey Could you explain the choice of stellar types rather than stars?

**Comment by BartlettAstro** [comment]
@adrianlucy You mentioned that MAST might like to see a "Stars" parent concept. Would this change address that need?

**Comment by adrianlucy** [comment]
Hi @BartlettAstro, thanks! Yes, from the MAST perspective, I support changing the prefLabel of 1634 to "Stars". I also support this from a broader perspective as someone who is still learning but does some vocabulary development work at MAST:

1. Consider a hierarchical vocabulary about animals. Which of the following would you do?

Option A: The parent concept of "Tiger" and "Lion" is "Cat types"
Option B: The parent concept of "Tiger" and "Lion" is "Cats"

Consider the logical endpoint of Option A: if "Tiger" has child concepts "Bengal tiger" and "Siberian tiger", now you might be tempted to change "Tiger" to "Tiger types". Now say "Bengal tiger" has child concepts "Shorthair Bengal tiger" and "Longhair Bengal tiger", maybe you are tempted to change "Bengal tiger" to "Bengal tiger types".  And so on down the tree for every concept that has children. The end result would be a vocabulary that is a bit incoherent in user-facing applications, because lots of concepts would have "types" appended for reasons that are opaque to everyone except someone looking at a map of the entire vocabulary.

Therefore, Option B is the way to go for the UAT. The fact that this is a parent concept that can encompass subtypes is implicit in the fact that the concept is in a hierarchical vocabulary; it doesn't need to be stated explicitly in the concept name. "Stellar types" is the "Cat types" of astronomy, and "Stars"/"Cats" is better.

2. From the perspective of MAST, or extracting an astronomical object type vocabulary from the UAT, or aligning the UAT to the Simbad/IVOA object type vocabulary, or using the UAT in any application with which you might want to search for stars: a "Stars" concept is necessary for good search filtering and navigation of the vocabulary hierarchy, and this is the place in which it makes the most sense, as a parent to all the different kinds of stars.

3. This is how the "Galaxies" tree works. The UAT doesn't have a "Galaxy types" concept, nor should it.

PS, the OP's point about brown dwarfs is also well taken, but also a can of substellar worms that I'm not sure you have the resources to deal with right now.

**Comment by adrianlucy** [comment]
As an aside, the "Stellar types" does also have two children of particular interest here, 2052 = Stellar evolutionary types and 2051 = Stellar spectral types. I get how these arose, and they might go some way towards explaining "Stellar types" itself. "Stellar evolutionary types" in particular sounds useful to the taxonomist who wants to group things together. 

Whether that extra layer of the hierarchy is useful to any end-user search application, whether the UAT should include concepts whose sole purpose is to play a role in taxonomical hierarchy, and whether the UAT is consistent about these practices in different parts of the hierarchy, is I think a much bigger question that the UAT should do some deep thinking about when you have more resources. I've been trained by Archive of Our Own volunteers to be hyper-skeptical of concepts that are there just for the sake of the vocabulary wrangler and aren't useful to the user, but I may not be considering all the angles and use cases, and I've sometimes found it hard to follow this advice at MAST. Maybe there's some role for a "search-relevant vocabulary" skos:ConceptScheme, or something like that.

In any case, 2052 and 2051 aren't necessarily a problem for MAST, since I'm thinking we can maybe automate something like "delete concept X and give its children to its parent" in our code logic without requiring action from the UAT.

Anyway tl;dr, changing 1634 "Stellar types" to "Stars" would be very well-received by MAST, and I don't see how it could cause problems for anyone else.

**Comment by katieefrey** [comment]
> Anyway all that is an aside... tl;dr, changing 1634 "Stellar types" to "Stars" would be very well-received by MAST, and I don't see how it could cause problems for anyone else.

First, I agree that a "stars" concept fits into the UAT.  I would caution _against_ updating the label for 1634, and recommend the option to create a new concept for "stars," replacing it in the hierarchy, and reorganizing as necessary.

I've mentioned this in other threads/emails to @BartlettAstro and @danielchivvis, but a concept's meaning should not change. To me "stars" and "Stellar types" are very different concepts. Also, the UAT concept "Stellar types" has been [attached to 20 papers in ADS](https://ui.adsabs.harvard.edu/search/fq=%7B!type%3Daqp%20v%3D%24fq_database%7D&fq_database=(database%3Aastronomy%20OR%20database%3Aphysics)&p_=0&q=((keyword%3A%221634%22)%20AND%20year%3A2018-2023)&sort=date%20desc%2C%20bibcode%20desc). I really think the usage of this concept is for "stellar types" and substituting "stars" would not work for them.

As an aside, the concept "Stars" was renamed to "Stellar types" sometime in 2015.  At this time, the UAT was in a large state of flux.  It wasn't published online, outside of this github repo, it wasn't being used by anyone.  We were still figuring a lot of stuff out, and concept URIs and IDs changed in each of the first four releases until we finally landed on the current scheme.  Here is the history of this concept:

|UAT version|release date|URI|Label|
|---|---|---|---|
|v.0.0.0 (beta)|Sept 4, 2014|http://purl.org/astronomy/uat/xl_en_b101b334|Stars|
|v1.0.0|Dec 23, 2015|http://www.altbibl.io/astronomy/uat/xl_en_b05c3958|Stellar types|
|v.1.1.0|July 10, 2016|http://astrothesaurus.org/uat/StellarTypes|Stellar types|
|v.2.0.0|Jan 31, 2017|http://astrothesaurus.org/uat/1634|Stellar types|

Changing a concept meaning from UAT beta through UAT v2.0.0, while still against best practices, simply did not have the same impact as making this change would have today.

So, I would recommend that 1634 _remain_ "Stellar types" or be deprecated, and a new concept be added to address "Stars."

The new "Stars" concept could probably generally replace 1634's location in the hierarchy.  (@BartlettAstro this would, however, have a large effect on the organizational structure of the UAT and would, IMO, push this to a "major" release)

> As an aside, the "Stellar types" concept does also have two children of particular interest here, 2052 = Stellar evolutionary types and 2051 = Stellar spectral types. I get how these arose, and they might go some way towards explaining "Stellar types" itself. "Stellar evolutionary types" in particular sounds useful to the taxonomist who wants to group things together.

Would "stellar spectral types" and "stellar evolutionary types" still fit under the "stars" concept?

**Comment by adrianlucy** [comment]
Good point, and that seems like a reasonable solution from the MAST perspective.

I will say, I think those papers may be using 1634 to refer to "Stellar typ**ing**", as a method, rather than "Spectral types" per se (see my complaint about "Cat types" above). Given current usage, I would suggest:
- Change 1634 to "Stellar typing", drop its parent/child relationships, and make it a child of "Stellar classification"
- Put "Stars" in the empty space left behind in the hierarchy, as Katie says

> Would "stellar spectral types" and "stellar evolutionary types" still fit under the "stars" concept?

"Stellar spectral types" isn't doing anything necessary in the hierarchy, which already has "Early-type stars", "Intermediate-type stars", and "Late-type stars" to group its children together. I think here again you could change it to "Stellar spectral typing", drop its relationships, and make it a child of "Stellar typing" again.

You could do a similar thing for "Stellar evolutionary types" and give its children to "Stars", but personally I would consider: deprecate it in favor of the existing "Stellar evolution", and give its children to "Stars".

Incidentally, in the children of "Stellar evolutionary types" I see some examples of deeper issues in the hierarchy: for example, why isn't "Asymptotic giant branch stars" a child of "Evolved stars"? Stuff like that which will take more work to untangle, but which would both help with transitive inference in search applications and help meet the taxonomist's urge to group things together.

**Comment by adrianlucy** [comment]
(Or deprecate "Stellar types" in favor of "Stellar classification" because they're being used in the same way and mean the same thing.)

**Comment by adrianlucy** [comment]
(Or instead of making "Stellar typ[ing]" a child of "Stellar classification", just deprecate it in favor of "Stellar classification" since it seems to mean the same thing and be used in the same way. Rest of my post the same except substitute "Stellar classification" for every subsequent instance of "Stellar typing".)

**Comment by adrianlucy** [comment]
This issue has been identified as a priority for Curation over the next 1.5 months, so what follows is a concrete proposal that attempts to synthesize everyone's perspectives. I also performed a quick investigation of concept usage and found:
- 26 AAS papers have author-tagged `Stellar types` (1634)
  - `Stellar types` has been used to describe ~three different things
    - "Stars" (only ~2 usages, so let's ignore this usage)
    - Differences between stellar types, ways in which a stellar type is different from other stellar types, or evolution between stellar types (~11 usages)
    - Techniques for classifying/finding stars as (a) stellar type(s) (~11 usages)
  - Nearly all usage (with ~1 exception) refers to stellar spectral types, suggesting that we could eventually aim to deprecate `Stellar types` (1634) in favor of `Stellar spectral types` (2051). However, since it's unclear to me whether ADS and other stakeholders have proper handling for concept deprecation/replacement, I will move that proposal to a separate issue.
- VizieR UAT tags to catalogs haven't used `Stellar types` (one of many intriguing gaps, e.g. `Symbiotic binary stars` has also not been used by VizieR)

Taking all that and the above discussion into account, here's my **proposal**:
1. New concept: create a new concept `Stars`
2. Remove altlabel: Remove the "Stars" altlabel from `Stellar types` (1634)
3. Add relationship: add `Stellar types` (1634) to the children of`Stellar classification` (1589)
4. Remove and add relationships: disconnect all direct children from `Stellar types` (1634) _except_ for `Stellar evolutionary types` (2052) and `Stellar spectral types` (2051). Then for all the concepts you just disconnected, make them direct children of the new concept `Stars`.
5. Remove and add relationships: remove the following concepts from the direct children of `Stellar evolutionary types` (2052) and instead make them direct children of `Evolved stars` (481): `Horizontal branch stars` (746), `White dwarf stars` (1799), `Yellow straggler stars` (1829), `Post-asymptotic giant branch stars` (2121), `Asymptotic giant branch stars` (2100), `Blue straggler stars` (168)
6. ~Remove and add relationship: Remove `Blue straggler stars` from the direct children of `Stellar evolutionary types` (2052) and instead make it a direct child of `Early-type stars` (430)~ - **[Edit: never mind, see Markus's comment. Added blue stragglers to the list in (5).]**
7. Remove and add relationships: Remove all _remaining_ direct children from `Stellar evolutionary types` (2052), and instead make them direct children of the new concept `Stars`
8. Remove relationships: Remove all direct children from `Stellar spectral types` (2051). No need to move them anywhere, they are all already grouped under the early/intermediate/late-type concepts (430, 818, 909).

After this change, `Stellar types`, `Stellar spectral types`, and `Stellar evolutionary types` are now intended to be abstract concepts encompassing classification (a technique) or the differences/evolution between stellar types (an area of study), detached in the polyhierarchy from concrete astronomical object types. In light of my analysis of usage in AAS papers, this seems to be necessary in order to be compliant with our curation policy: papers tagged with e.g. `M stars` are not necessarily _about_ `Stellar types` in the way that `Stellar types` is being understood by users, so `M stars` cannot be a descendant of `Stellar types`:
> Each created or altered concept must be always a type of its parents or always a component of its parents, and/or data and papers tagged with the concept must always also contain its parents. “Always” means “in all astronomical contexts,” including solar system science, planetary science, and laboratory astrophysics.

**Comment by msdemlei** [comment]
On Wed, Feb 04, 2026 at 10:59:40AM -0800, Adrian Lucy wrote:
> 1. New concept: create a new concept `Stars`

Having to create such a fundamental concept on such a mature resource
like the UAT is a shame, but everything less intrusive got stalled,
too, and this is at least clean and clear.

> 4. Remove and add relationships: disconnect all direct children
> from `Stellar types` (1634) _except_ for `Stellar evolutionary
> types` (2052) and `Stellar spectral types` (2051). Then for all the
> concepts you just disconnected, make them direct children of the
> new concept `Stars`.

If I hadn't witnessed the drama so far, I'd say "OMG please don't"
now.

Having witnessed it, I'd say in our commit message we should stress
that this kind of open-heart surgery must be reserved for
emergencies.  An astronomy thesaurus without a clear concept of
"Star" is an emergency; almost anything else isn't.

Meanwhile: yes, that's what we ought to do.

> 6. Remove and add relationship: Remove `Blue straggler stars` from
> the direct children of `Stellar evolutionary types` (2052) and
> instead make it a direct child of `Early-type stars` (430)

Hm.  I'll not make a fuss, but I'd rather like to see them
under evolved-stars (which they are).  Calling them early type, on
the other hand, doesn't really capture why they're special, and blue
stragglers can be G or F, which few people would call these "early",
at least outside of globular clusters.

Other than that, this is the best-thought-out plan I've seen for the
*#stars problem.  I'd be happy to support you in getting it into 6.1
in whatever way I can.

**Comment by adrianlucy** [comment]
Thanks for the review!

> this kind of open-heart surgery must be reserved for emergencies

I'm overall not _very_ worried personally about en-masse changes limited to broader/narrower relationships, especially in this phase where downstream application use of the hierarchy is fairly minimal. But yes, when coupled with the whole "creating a concept for Stars" thing this is a uniquely heightened level of drama. Anyway, I'll add that broader strategic discussion to the curation subcommittee's docket.

> I'd rather like to see [blue stragglers] under evolved-stars

Thanks, I'll edit my post to move them under evolved stars and cut my proposed connection to early-type.

**Comment by adrianlucy** [comment]
Closing this issue; this change will be included in the next release.

Specifically, the change to be made is that described in my modified Feb 4 post. **Note that this differs from the title of this git issue, which we moved away from.**

> Taking all that and the above discussion into account, here's my **proposal**:
> 
>     1. New concept: create a new concept `Stars`
> 
>     2. Remove altlabel: Remove the "Stars" altlabel from `Stellar types` (1634)
> 
>     3. Add relationship: add `Stellar types` (1634) to the children of`Stellar classification` (1589)
> 
>     4. Remove and add relationships: disconnect all direct children from `Stellar types` (1634) _except_ for `Stellar evolutionary types` (2052) and `Stellar spectral types` (2051). Then for all the concepts you just disconnected, make them direct children of the new concept `Stars`.
> 
>     5. Remove and add relationships: remove the following concepts from the direct children of `Stellar evolutionary types` (2052) and instead make them direct children of `Evolved stars` (481): `Horizontal branch stars` (746), `White dwarf stars` (1799), `Yellow straggler stars` (1829), `Post-asymptotic giant branch stars` (2121), `Asymptotic giant branch stars` (2100), `Blue straggler stars` (168)
> 
>     6. [STEP 6 REMOVED, PLEASE IGNORE]
> 
>     7. Remove and add relationships: Remove all _remaining_ direct children from `Stellar evolutionary types` (2052), and instead make them direct children of the new concept `Stars`
> 
>     8. Remove relationships: Remove all direct children from `Stellar spectral types` (2051). No need to move them anywhere, they are all already grouped under the early/intermediate/late-type concepts (430, 818, 909).
> 
> 
> After this change, `Stellar types`, `Stellar spectral types`, and `Stellar evolutionary types` are now intended to be abstract concepts encompassing classification (a technique) or the differences/evolution between stellar types (an area of study), detached in the polyhierarchy from concrete astronomical object types. In light of my analysis of usage in AAS papers, this seems to be necessary in order to be compliant with our curation policy: papers tagged with e.g. `M stars` are not necessarily _about_ `Stellar types` in the way that `Stellar types` is being understood by users, so `M stars` cannot be a descendant of `Stellar types`:
